code/game/machinery/computer/crew.dm

```diff
@@ -279,7 +279,7 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 				entry["ijob"] = jobs[trim_assignment]
 
 		// SKYRAT EDIT BEGIN: Checking for robotic race
-		if (issynthetic(tracked_human))
+		if (issynthetic(tracked_human) || isprotean(tracked_human)) // BUBBER EDIT - Added "|| ifsprotean(tracked human)"
```

Comment:
Since this line is already within a SKYRAT EDIT BEGIN block you can just change the begin and end to say BUBBER instead, the comment on this line specifically is unneeded